adapters/net/encode.go

```diff
@@ -0,0 +1,38 @@
+package net
+
+import (
+	"net"
+	"reflect"
+
+	"github.com/segmentio/objconv"
+)
+
+// EncodeTCPAddr encodes the net.TCPAddr value in v as a string using e.
+func EncodeTCPAddr(e objconv.Encoder, v reflect.Value) error {
```

Comment:
Doesn't seem necessary to expose these functions in the public API. I think having a single `Adapter` implementation exposed should suffice?

```
func TCPAddrAdapter() objconv.Adapter {
       return objconv.Adapter{
		Encode: encodeTCPAddr, // hidden EncodeTCPAddr
 		Decode: decodeTCPAddr, // hidden DecodeTCPAddr
 	}
}
```